spec/features/diy/new_diy_client_spec.rb

```diff
@@ -9,6 +9,10 @@
 
   scenario "a new client files through TaxSlayer" do
     allow(MixpanelService).to receive(:send_event)
+    visit "/diy"
+    expect(page).to have_selector("h1", text: "File taxes on your own for free!")
+    expect(page).to have_selector("p", text: "We provide free access to an online tax prep site called TaxSlayer. We'll send you a link to create your own TaxSlayer login, so you can file on your own.")
+    click_on "Continue"
```

Comment:
If you are doing `click_on` here, then I recommend not doing `visit` below. Personally I would not cover a landing page with a feature spec, but it's OK with me if we do!